source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/products/index.tsx

```diff
@@ -29,21 +29,34 @@ interface IOwnProps {
   isEmptyWorkflowProjectUrl: boolean;
 }
 
+interface IPendingUpdates {
+  [itemId: string]: boolean;
+}
+
 type IProps =
   & IOwnProps
   & i18nProps
   & IDataActionsProps;
 
 class Products extends React.Component<IProps> {
+  pendingUpdates: IPendingUpdates;
```

Comment:
idk what conventions you prefer, or whatever, but I guess just fyi, this can be one-lined to

```ts
pendingUpdates: IPendingUpdates = {};
```

and then the constructor can go away

Reply:
i'll one line it.